pt-BR/mail/chrome/messenger/importDialog.dtd

```diff
@@ -36,26 +36,26 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!--
-LOCALIZATION NOTE : 'Communicator 4.x' is the used for previous versions of 
-Netscape Communicator, Please translate using the brandname in respective 
+LOCALIZATION NOTE : 'Communicator 4.x' is the used for previous versions of
+Netscape Communicator, Please translate using the brandname in respective
 languages for Netscape Communicator 4 releases.
-LOCALIZATION NOTE : Do not translate any of the occurrences of the word 
+LOCALIZATION NOTE : Do not translate any of the occurrences of the word
 "&brandShortName;" below.
 -->
 
 <!ENTITY importDialog.windowTitle "Assistente de importação">
-<!ENTITY importDialog.allTitle    "Import everything">
-<!ENTITY importDialog.mailTitle   "Mail">
-<!ENTITY importDialog.addressbookTitle "Address Books">
-<!ENTITY importDialog.settingsTitle "Settings">
+<!ENTITY importDialog.allTitle    "Importar tudo">
+<!ENTITY importDialog.mailTitle   "E-mails">
+<!ENTITY importDialog.addressbookTitle "Catálogos de endereços">
+<!ENTITY importDialog.settingsTitle "Configurações">
 
 <!ENTITY  window.width            "40em">
 <!ENTITY  window.macWidth         "45em">
 
 <!ENTITY importTitle.label        "Assistente de importação do &brandShortName;">
-<!ENTITY importDesc.label         "Import Mail, Address Books and Settings from other programs">
+<!ENTITY importDesc.label         "Importar e-mails, catálogos de endereços e configurações de outros programas">
 
-<!ENTITY importDescription.label  "This wizard will import mail messages, address book entries, and/or preferences from other mail programs and common address book formats into &brandShortName;.">
+<!ENTITY importDescription.label  "Este assistente importará para o &brandShortName; e-mails, entradas de catálogos de endereços e/ou preferências de outros programas de e-mail e formatos comuns de catálogos de endereços.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY importDescription.label  "Este assistente importará para o &brandShortName; mensagens, entradas de catálogos de endereços e/ou preferências de outros programas de correio eletrônico e formatos comuns de catálogos de endereços.">